.github/workflows/test_neo4j.yml

```
@@ -14,6 +15,8 @@ env:
   RUNTIME__LOG_LEVEL: ERROR
 
 jobs:
+  if: ${{ github.event.label.name == 'run-checks' }}
```

actionlint

"steps" section is missing in job "if"

(syntax-check)